### PR TITLE
Project health dashboard

### PR DESCRIPTION
**Pull request by czue**
probably easiest to review all at once.

image

This is behind a feature flag so should be safe to merge in anytime after code review.

@biyeun buddy cc @sravfeyn since this involves the MALT and @amstone326

### CODE REVIEW

File: corehq/apps/hqwebapp/templatetags/hq_shared_tags.py [hunk 1]
```diff
@@ -347,3 +347,13 @@ def render(self, context):
         output = self.nodelist.render(context)
         context[self.varname] = output
         return ''
+
+@register.simple_tag
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/templatetags/hq_shared_tags.py]: E302 expected 2 blank lines, found 1

**Comment by snopoke** [comment]
LGTM

File: corehq/apps/reports/standard/project_health.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
+from collections import namedtuple
+import datetime
+from django.db.models import F
+from django.utils.translation import ugettext_noop
+from corehq.apps.data_analytics.models import MALTRow
+from corehq.apps.reports.standard import ProjectReport
+from corehq.apps.style.decorators import use_nvd3, use_bootstrap3
```
Comment (dimagimon) [inline corehq/apps/reports/standard/project_health.py]: F401 'use_bootstrap3' imported but unused

**Comment by czue** [comment]
:white_check_mark:

**Comment by czue** [comment]
@dimagi/product not sure whether you want to figure out how to UAT/release this. My plan was to announce internally but didn't have a plan beyond that

**Comment by benrudolph** [comment]
:green_heart: cool idea!

**Comment by amsagoff** [comment]
@czue added it to our sync agenda to chat through how to productize!

**Pull request merged**